### PR TITLE
ACMS-1505: Remove module acquia_telemetry dependency from common.

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**

Fixes #NNN

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_common/acquia_cms_common.info.yml [hunk 1]
```diff
@@ -5,7 +5,6 @@ type: module
 core_version_requirement: ^9
 dependencies:
   - drupal:acquia_purge
-  - drupal:acquia_telemetry
```
Comment (apathak18) [inline modules/acquia_cms_common/acquia_cms_common.info.yml:8]: Is it on purpose to keep the module in composer.json of aquia_cms_common

Screenshot 2022-10-20 at 7 04 08 PM

**Review by apathak18 — APPROVED** [review]
Sorted.. Looks good to me.

**Review by rajeshreeputra — APPROVED** [review]
Changes looks good.

File: modules/acquia_cms_common/acquia_cms_common.info.yml (hunk 1 above)
Comment (rajeshreeputra) [reply modules/acquia_cms_common/acquia_cms_common.info.yml:8]: @apathak18 - If it is removed from composer then existing sites breaks as acquia_telemetry is installed and missing.

**Pull request closed without merging**